pombola/south_africa/templates/south_africa/local_rep_list_item.html

```diff
@@ -0,0 +1,84 @@
+{% load staticfiles %}
+{% load thumbnail %}
+
+<li id="{{ person_data.element_id }}">
+    {# Display a photo if one is available... #}
+    {% if person_data.person %}
+      <a href="{{ person_data.person.get_absolute_url }}">
+      {% thumbnail person_data.person.primary_image "58x58" crop="center" as im %}
+        <img class="constituency-office-mp-photo" src="{{ im.url }}" width="{{ im.width }}" height="{{ im.height }}"/>
+      {% empty %}
+        <img class="constituency-office-mp-photo" src="{% static 'images/person-90x90.jpg' %}" width="58" height="58" />
+      {% endthumbnail %}
+    {% else %}
+        <img class="constituency-office-mp-photo" src="{% static 'images/person-90x90.jpg' %}" width="58" height="58" />
+    {% endif %}
+
+    {# The representative's name #}
+      <p class="constituency-office-mp-name">{{ person_data.name }}</p>
+    {% if person_data.person %}
+      </a>
+    {% endif %}
+
+    {# This person's position (either as an MP, an MPL or a ward councillor #}
+    {% if not skip_positions %}
+      <ul class="unstyled-list">
+      {% for position in person_data.positions %}
+        {% if position.title.name == 'Ward Councillor' %}
```

Comment:
Perhaps better to have a lookup of preserved titles?

Pass in a `preserved_titles` set `{'Ward Councillor', 'Constituency Contact'}` and do '{% if if position.title.name in preserved_titles %}<li>position.title.name</li>{% else %}'